src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig

```diff
@@ -232,6 +232,18 @@
     </div>
 {%- endblock radio_row %}
 
+{# Support #}
+
+{% block empty_row -%}
+    {%- set translation_domain = translation_domain ?: 'sf_form' -%}
+    {%- set attr = attr|merge({class: (attr.class|default('') ~ ' form-control')|trim}) %}
+    <div {{ block('widget_container_attributes') }}>
+        <p>
+            <em>{{- translation_domain is same as(false) ? empty_view : empty_view|trans({}, translation_domain) -}}</em>
```

Comment:
I agree, I did not even think about it when I added it naturally.

Because, it may be unexpected to see that string standing for a missing expectation.

The `emphasis` clearly mark the informational sense of such string IMHO.